server/src/test/java/io/druid/server/coordinator/DruidCoordinatorConfigTest.java

```diff
@@ -42,7 +42,6 @@ public void testDeserialization() throws Exception
     Assert.assertEquals(new Duration("PT300s"), config.getCoordinatorStartDelay());
     Assert.assertEquals(new Duration("PT60s"), config.getCoordinatorPeriod());
     Assert.assertEquals(new Duration("PT1800s"), config.getCoordinatorIndexingPeriod());
-    Assert.assertFalse(config.isMergeSegments());
```

Comment:
Suggest adding javadoc to test class indicating where the tests for the merge configs are located.